src/dmd/backend/cod3.d

```diff
@@ -1346,11 +1341,20 @@ version (MARS)
  */
 regm_t allocretregs(tym_t ty, type *t, tym_t tyf, reg_t *reg1, reg_t *reg2)
 {
+    //printf("allocretregs()\n");
     tym_t ty1 = ty;
     tym_t ty2 = TYMAX;
 
     *reg1 = *reg2 = NOREG;
 
+    /* Broken for EX_WIN64 and 16 bit code
+     * Broken for complex numbers with EX_windos - should return in STO1
+     */
```

Comment:
```suggestion
    /* This uses the old approach the backend used to pass parameters
      * (pre dlang/dmd#10200 which changed parameter passing to follow the Itanium ABI)
      * when generating 16 bits code on Windows, and is only used by DMC, not DMD.
      */
```

This is, I hope, a little more descriptive to the average reader.

Reply:
I rewrote the comments. It is time to add links to PRs, that's what git blame is for. The entire back end supports 16 bit code generation, putting in comments explaining why 16 bit code gen is there in this spot doesn't make sense.